### ISSUE TITLE
[REQ] Consider creating an Elementary-style theme

### ISSUE THREAD

**Issue by ripefig** [n4n0GH/hello#47]
**Describe the feature you'd like to see**
Elementary kvantum/qt-style and window decorations, plasma and color scheme.

**Additional context**
I know you want this to be a mac theme - this is just a suggestion.

Elementary has a very polished and unique visual style which is unfortunately not available in QT land. I think the audience for it is wider than for yet another Mac clone, since Elementary is much more subdued in terms of the design. Mac is very opinionated and loud. I have yet to find anyone who doesn't like Elementary design. Most people seem to think its the best design on Linux, which isn't surpising since the whole project was created for the sake of a theme.

I think most people who like Aqua ui will also like Elementary. The reverse however is less true (I like Elementary but hate Aqua).

Your window border is already very close to Elementary decorations. Furthermore, Elementary has a wide range of "supporting" third-party themes, including themes for web browsers, text editors, general gtk theme, icon themes for libreoffice etc.  There also supplementary elementary free-desktop icons and the Newaita theme (the most complete icon theme on Linux today by far), which would go well together with an Elementary widget theme.

Furthermore, an Elementary QT theme could uniquely bridge the gap between Gnome, Elementary and KDE. I think that a theme must bridge multiple platforms to be successful, and there are very few themes that do this, therefore they are kind of useless since they don't cover half your applications.

There is currently no theme that does this except Breeze and it doesn't do it too well. Elementry itself is notoriously hard to theme, so it makes sense to take Elementary theme as a given and then just make normal GTK and QT apps conform. This would also greatly benefit QT apps running on GTK-based desktops.

Curious to hear what you think about this idea?

**Comment by n4n0GH** [comment]
Considering that elementaryOS was started by someone who made a fairly popular icon set for Linux many many years ago with a style that was nice back then and who turned that entire style into a philosophy of an entire operating system which looks like it was stuck in the early 2000's / 2010's of design hell AND which was heavily inspired not only by Mac OSX but also Apple's strict HIG, I find it quite ironic to find a request like that.

TL;DR of this REQ is that I won't do it because I don't want to validate or enable other developers that call their userbase "thieves" if they don't donate to their project.

So here's my personal take on your request, no hard feelings.

> I think the audience for it is wider than for yet another Mac clone, since Elementary is much more subdued in terms of the design.

Hard doubt. If there was a great desire for an eOS theme on KDE, someone would've already made one.

> Mac is very opinionated and loud.

eOS is the most restrictive desktop I have ever used. I agree that they have some pretty things going on and their strict guidelines make sense to keep a consistent interface. But not allowing users to officially theme the desktop or even have certain window control buttons because someone once said "nope, that's not happening" is kinda not what I want on a Linux distribution. Yes, there are third party applications like there are for Gnome, but saying macOS is opinionated and not saying the same for eOS is kinda hypocritical.

> Your window border is already very close to Elementary decorations.

That is because both eOS and my window decoration are heavily influenced and inspired by macOS/OSX.

> Furthermore, an Elementary QT theme could uniquely bridge the gap between Gnome, Elementary and KDE.

I really don't see how this would be important. eOS is a niche operating system that doesn't hold any weight and likes to keep it's tools to itself.

**Comment by ripefig** [comment]
Gotcha, but if you don't mind I'll just reply:

Their theme is a modernized version of older Aqua, just like current Aqua. They just took it in a different direction and I think most people would say it looks more "classic" but not in a bad way. "Flat" design with bright colors is a trend that's at least a decade old, so it's also arguably dated - and you can see platforms like Windows now moving away from it.

Thereare some hacky Elementary Kvantum themes on store.kde.org, but I don't think you can make any conclusions based on what's available there. There are few themes available for KDE and most of them aren't really usable.

We probably agree that Elementary is doing some things wrong as a platform, but their theme is often considered the best on Linux and it's GPL, unlike Aqua. I think it's better to validate Elementary than a ruthless corporation like Apple, no?

EoS is certainly marginal and doesn't care about interop, but you could say that about Gnome too. Every platform in Linux world is marginal. If you get rid of all the marginal platforms, you'll have nothing left.

**Comment by n4n0GH** [comment]
I don't mind, reply as often as you like :-)

> Their theme is a modernized version of older Aqua

It literally is not. By the time they released their eOS it already looked way outdated compared to the OS X versions at the time. And they never caught up, they never wanted to because Dan is fixed in his mindset to have supplied god's gift to the UI and UX world.

> "Flat" design with bright colors is a trend that's at least a decade old

True, but it has evolved and never stayed the way it was like ten years ago.

> and you can see platforms like Windows now moving away from it.

Are we seeing the same Windows versions? All Microsoft is doing is adding blur and glass effects to otherwise _very_ flat interfaces. This is one of the evolutions of flat design. OT and personal opinion: I think that a mixture of skeuomorphism and flat will be the next trend in design.

> We probably agree that Elementary is doing some things wrong as a platform, but their theme is often considered the best on Linux

You're actually the first person that tells me it's regarded as one of the best themes. By whom? None of the design and Linux circles I'm in holds it in any regard. In fact, most laugh at it.

> I think it's better to validate Elementary than a ruthless corporation like Apple, no?

You don't have to validate either of those. If someone feels arrogant enough to shit on their users by calling them thieves then I don't want to have anything to do with them. Just because some other company does other stuff that's bad, doesn't suddenly increase the validation of the aforementioned. That's a whataboutism.

> EoS is certainly marginal and doesn't care about interop, but you could say that about Gnome too.

Gnome is not at all marginal. It is the default desktop for many distributions and serves as the base for newer Ubuntu releases since Canonical dropped Unity. Realistically, Gnome and KDE shape the desktop space, and for a good reason: they don't dictate what distro you should run to use their desktop environments.

> If you get rid of all the marginal platforms, you'll have nothing left.

I agree, but this has nothing to do with me not wanting to "clone" something I personally think doesn't look aesthetic, nor do I agree with how the developers treat their userbase.

**Comment by ripefig** [comment]
> OT and personal opinion: I think that a mixture of skeuomorphism and flat will be the next trend in design.

It seems to me that elemtary (and to some extent Gnome) is doing that. It's not skeumorphic in the sense of Aero or older IOS/Mac. The icons aren't photorealistic or overly detailed either. So Elementary is pretty minimal but not synthetically flat like Metro or Material. Windows is sort of moving in a similar direction (the shapes look sculpted but the interface is still minimalist) if this picture is any guide:

![image](https://user-images.githubusercontent.com/45201036/70576481-16e20680-1bba-11ea-95b4-ecb8709e3240.png)

So to me Elementary and Gnome look more modern than any other platform precisely for the reason you mentioned. I think the argument boils down to whether or not something looks flashy and synthetic as opposes to subdued and "natural". EoS and to a lesser extent Gnome use more subdued/warm colors and don't use glass effects, so some say they  "look dated." KDE's design is pretty dated, haphazard and incoherent, but since it uses cold flashy colors and blur effects it's considered "modern." To me it's just good vs bad design really.

The embrace of flashy colors and blur by the big companies has more to do with trying to catch consumers short-term attention than good design. Looks vulgar and distracting to me personally.

**Comment by n4n0GH** [comment]
> It seems to me that elemtary (and to some extent Gnome) is doing that. It's not skeumorphic in the sense of Aero or older IOS/Mac. The icons aren't photorealistic or overly detailed either. 

So then it's not skeuomorphic.

> So Elementary is pretty minimal but not synthetically flat like Metro or Material.

eOS looks like an early 2000's version of Xfce to be honest. It [looked like that when it started](https://news.softpedia.com/images/extra/LINUX/large/elementaryos01-large_003.jpg) and it [still looks like it](https://nerdzoom.de/wp-content/uploads/2019/12/Elementary_OS_5.1_Hera_-_files1.jpg).

> On fluent design

How does anything look sculpted? The window frames will look as flat as they do now on Windows 10, icons will be the same monochrom line-art they are like now and probably resemble a lot of what Breeze icons are currently looking like.

> So to me Elementary and Gnome look more modern than any other platform precisely for the reason you mentioned.

What reason? Gnome is producing as much of a flat interface as Windows 10, only difference is their massive headerbar that uses a gradient and highlight lines to create the illusion of raised surfaces.

> EoS and to a lesser extent Gnome use more subdued/warm colors and don't use glass effects, so some say they "look dated."

Uh, are we talking about the same thing? Gnome uses a beige like color for their headerbar, but otherwise relies on black and white for windows and their top panel. eOS uses mostly grey to render their windows. And it's not a warm grey, but a grey with more blue than red/yellow hues in it. That's cold, not warm. I don't know anyone that says the two look dated for the choice of color or non-use of blur/glass effects. Only reason people say eOS looks dated is because it looks like it's an Xfce mod (which by the way used to look [like this way before eOS was a thing](https://bluesabre.org/content/images/size/w2000/2017/12/xfce_pulseaudio_plugin_0.3.4.png)).

> KDE's design is pretty dated, haphazard and incoherent

The design is pretty conservative, I agree, but it's the lowest common denominator in terms of picking up most users with the little knowledge they have about computing and instantly enabling them to get work done without having to learn a new interface. And in regards to the incoherence I can only say that the developers and community are currently working like crazy to make it the most coherent desktop experience available on Linux.

> To me it's just good vs bad design really.

eOS put form over function on every aspect to create a "shiny" interface. Sometimes that works out, sometimes usability is impaired. Gnome did it even worse to the point where you _have_ to install third party plugins to regain functionality of a modern desktop where people actually work instead of just consuming Facebook or Youtube. KDE on the other hand put more thought into how users actually use their machines which led to a clusterfuck of menu options and choices in terms of how you can use a single application. It's not ideal either but personally I'd rather have all options available and being able to turn them off myself than having to rely on a plugin I am not sure will receive updates in the future.

> The embrace of flashy colors and blur by the big companies has more to do with trying to catch consumers short-term attention than good design. Looks vulgar and distracting to me personally.

Well, Apple started pushing this trend with iOS and later macOS software. People liked that additional immersion effect they got out of this and other developers started doing similar things. I don't see how this is bad. Of course it has to be done tasteful. But hey, KDE allows you to turn it off entirely ;-)

> I don't think you're validating anything by copying a theme

But that is exactly what validation is: "your software is so amazing I'm going to copy it". After all, imitation is the sincerest form of flattery.

**Comment by ripefig** [comment]
> But that is exactly what validation is: "your software is so amazing I'm going to copy it". After all, imitation is the sincerest form of flattery.

Yeah but that means you're validating Apple and I'm not sure how that's better. However, after myopic comments like [this](https://twitter.com/DanielFore/status/991928331632238592) from Elementary's founder, I can see why many people really hate these guys. The guy created an icon set and platform with a few toy apps and he's calling Mac OS a "dead platform", just unbelievable. 

Anyway, I still think it's a good idea to copy their theme but we can agree to disagree.

**Comment by n4n0GH** [comment]
> but we can agree to disagree

:-)
